setup.py

```diff
@@ -93,9 +93,9 @@ def fullsplit(path, result=None):
         "tools/eoxserver-preprocess.py"
     ],
     install_requires=[
-        'django<3',
+        'django<5',
```

Comment:
We use Django 3.2 in the tests, So I guess we should use `django<4` here

Reply:
changed to django<4